include/zephyr/drivers/dali.h

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (c) 2025 by Sven Hädrich <sven.haedrich@sevenlab.de>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef DRIVERS_DALI_INCLUDE_DALI_H_
+#define DRIVERS_DALI_INCLUDE_DALI_H_
```

Comment:
```suggestion
#ifndef ZEPHYR_INCLUDE_DRIVERS_DALI_H_
#define ZEPHYR_INCLUDE_DRIVERS_DALI_H_
```

Same at the closing comment.